Bender.lock

```diff
@@ -7,8 +7,8 @@ packages:
     dependencies:
     - common_cells
   axi:
-    revision: 9402c8a9ce0a7b5253c3c29e788612d771e8b5d6
-    version: 0.39.3
+    revision: 587355b77b8ce94dcd600efbd5d5bd118ff913a7
+    version: 0.39.4
```

Comment:
Remember 0.39.4 has a killer bug, I saw in the upstream the release was modified to be the same as 0.39.5, but here we are pointing to the wrong commit (see [here](https://github.com/pulp-platform/axi/commits/v0.39.4/)].